### ISSUE TITLE
Docker container doesn't work

### ISSUE THREAD

**Issue by hacktek** [v0idp/Mellow#80]
I can't get the docker container to start.

Running the command results in an empty host volume and no settings.json (is that expected?).

I manually put the file there with wget from https://raw.githubusercontent.com/v0idp/Mellow/master/data/settings.json and the container started.

Logs:

```
Error: Cannot find module '/usr/src/app/data/settings.json'


Require stack:


- /usr/src/app/src/util.js


- /usr/src/app/src/WebServer.js


- /usr/src/app/src/index.js


    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)


    at Module.Hook._require.Module.require (/usr/local/lib/node_modules/pm2/node_modules/require-in-the-middle/index.js:51:29)


    at require (internal/modules/cjs/helpers.js:74:18)


    at Object.<anonymous> (/usr/src/app/src/util.js:5:19)


    at Module._compile (internal/modules/cjs/loader.js:955:30)


    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)


    at Module.load (internal/modules/cjs/loader.js:811:32)


    at Function.Module._load (internal/modules/cjs/loader.js:723:14)


    at Module.require (internal/modules/cjs/loader.js:848:19)


    at Module.Hook._require.Module.require (/usr/local/lib/node_modules/pm2/node_modules/require-in-the-middle/index.js:70:39)


    at require (internal/modules/cjs/helpers.js:74:18)


    at Object.<anonymous> (/usr/src/app/src/WebServer.js:6:32)


    at Module._compile (internal/modules/cjs/loader.js:955:30)


    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)


    at Module.load (internal/modules/cjs/loader.js:811:32)


    at Function.Module._load (internal/modules/cjs/loader.js:723:14) {


  code: 'MODULE_NOT_FOUND',


  requireStack: [


    '/usr/src/app/src/util.js',


    '/usr/src/app/src/WebServer.js',


    '/usr/src/app/src/index.js'


  ]


}


2020-01-27T18:27:44: PM2 log: App [index:0] exited with code [1] via signal [SIGINT]


2020-01-27T18:27:44: PM2 log: Script /usr/src/app/src/index.js had too many unstable restarts (16). Stopped. "errored"


2020-01-27T18:27:45: PM2 log: 0 application online, retry = 3


2020-01-27T18:27:47: PM2 log: 0 application online, retry = 2
```

**Comment by v0idp** [comment]
Can you please try and re-deploy the container from scratch and see if it solves the issue ?

**Comment by v0idp** [comment]
Fixed this in 2ce0225
